src/main/java/kr/allcll/seatfinder/AuthInterceptor.java

```diff
@@ -0,0 +1,49 @@
+package kr.allcll.seatfinder;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.util.Arrays;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.ModelAndView;
+
+@Component
+public class AuthInterceptor implements HandlerInterceptor {
+
+    private static final String TOKEN_KEY = "token";
+
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+        if (isNotExistToken(request)) {
+            ThreadLocalHolder.SHARED_TOKEN.set(TokenProvider.createToken());
+            return true;
+        }
+        ThreadLocalHolder.SHARED_TOKEN.set(findTokenAtCooke(request));
+        return true;
```

Comment:
다음을 제안해 보고 싶은데요! 기존 `isNotExistToken`메서드의 반환값을 반대로 했어요. 메서드 명에 부정 표현을 지우고, 메서드 반환 값을 반대로 뒤집는다면, 읽을 때 부정 표현에 대한 계산을 하지 않아도 되기 때문에 더 편하다고 생각해요. 어떻게 생각하시나요? 

```suggestion
        if (isTokenExist(request)) {
            ThreadLocalHolder.SHARED_TOKEN.set(findTokenAtCooke(request));
            return true;
        }
        ThreadLocalHolder.SHARED_TOKEN.set(TokenProvider.createToken());
        return true;
```

Reply:
처음에 제안 주신 대로 했다가 isNotExist로 바꿨는데요,
그 이유는 강조하고 싶은게 '존재 하지 않는 상황'이어서 였어요!
로직 상 토큰이 존재하지 않을 때 유의미한 변경사항(token을 새로 발급받는다거나, response에 cookie를 추가한다거나)이 생긴다고 생각해서요...
어떻게 생각하세요?
`!isNotExistToken`이 쓰이는 경우가 있었다면 제안해 주신대로 했을 것 같은데, 모든 변경사항이 생기는 로직은 '토큰이 없을 때'였어서요.. 일부러 그렇게 했습니다
의견 주세용

Reply:
아하.. 💡 그런 의도가 있었군요! 저도 좋다고 생각해요. 메서드명만 좀더 가독성 있게 수정해 보면 어떨까요? `hasNoToken`, `isTokenAbsent` 등이 떠올라요